src/parameters/all_parameters.cpp

```diff
@@ -58,6 +58,28 @@ void AllParameters::declare_parameters (dealii::ParameterHandler &prm)
                       dealii::Patterns::Bool(),
                       "Use other boundary conditions by default. Otherwise use periodic (for 1d burgers only");
 
+    prm.declare_entry("use_energy", "false",
+                      dealii::Patterns::Bool(),
+                      "Not calculate energy by default. Otherwise, get energy per iteration.");
+
+    prm.declare_entry("use_L2_norm", "false",
+                      dealii::Patterns::Bool(),
+                      "Not calculate L2 norm by default (M+K). Otherwise, get L2 norm per iteration.");
+
+    prm.declare_entry("use_classical_Flux_Reconstruction", "false",
+                      dealii::Patterns::Bool(),
+                      "Not use Classical Flux Reconstruction by default. Otherwise, use Classical Flux Reconstruction.");
```

Comment:
I don't exactly understand the option.

Reply:
This only reconstructs the surface. So for nonlinear problems, its a good way of showing that ESFR with split forms is unstable unless you apply our volume reconstruction to the split form.